doc/src/general-rules.md

```diff
@@ -6,8 +6,6 @@
 and we kept a copy of the Rust standard library inside the `library/` folder that shall be used as the verification target for all our challenges.
 We will periodically update the `library/` folder to track newer versions of the [official Rust standard library](https://github.com/rust-lang/rust/).
 
-**NOTE:** This work is not officially affiliated, or endorsed by the Rust project or Rust Foundation.
```

Comment:
Should we instead just have a note that it *is* endorsed by the Rust Foundation (and perhaps link to their blog post)?